public/widget/discrepancy-note/Dn.js

```diff
@@ -583,7 +583,7 @@ Comment.prototype._renderHistory = function() {
     var $more = over3 > 0 ? $( '<tr><td colspan="4"><span class="over">+' + over3 + '</span>' +
         '<button class="btn-icon-only btn-more-history"><i class="icon"> </i></button></td></tr>' ) : $();
     this.$history.find( 'table' ).empty()
-        .append( '<thead><tr><td></td><td>' + historyText + '</td><td>' + user + '</td><td>' + clock + '</td></tr></thead>' )
+        .append( '<thead><tr><td colspan="2"><b>' + historyText + '</b></td><td>' + user + '</td><td>' + clock + '</td></tr></thead>' )
```

Comment:
Update to use <strong> instead of <b> and change it back to 2 column on latest update